Dockerfiles/agent/cont-init.d/50-ecs-managed.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Quick exit unless ECS managed instances are explicitly enabled
+if [[ "${ECS_MANAGED_INSTANCES:-}" != "true" ]]; then
+    exit 0
+fi
+
+# Require DD_ECS_DEPLOY_MODE=sidecar
+if [[ "${DD_ECS_DEPLOYMENT_MODE:-}" != "sidecar" ]]; then
+  exit 0
+fi
+
+# Set a default config for ECS Managed Instances if found
+# Don't override /etc/datadog-agent/datadog.yaml if it exists
+if [[ ! -e /etc/datadog-agent/datadog.yaml ]]; then
+    ln -s  /etc/datadog-agent/datadog-ecs.yaml \
+           /etc/datadog-agent/datadog.yaml
```

Comment:
Consider making the symlink _relative_:
```suggestion
    ln -s datadog-ecs.yaml /etc/datadog-agent/datadog.yaml
```
Why:
- most Linux packages use relative symlinks when source and target are in the same directory,
- even if users bind-mount the config directory, the symlink is created inside the container at the fixed path. Otherwise, if someone extracts the container filesystem (inspection, etc.), the absolute path would point to the original/hardcoded location (`datadog.yaml` -✗> `/etc/datadog-agent/datadog-ecs.yaml`).